### DISCUSSION TITLE
React version of this projet.

### DISCUSSION THREAD

**Discussion by soulsam480** [cawa-93/vite-electron-builder#643] [Forks]
Hii @cawa-93 amazing repo. This works great with vue. Can we have a port of this project with `React`. That will be great 😋⚛

**Comment by cawa-93** [comment]
Of course you can.  if you have need - make a fork and change it.  I think it should be quite simple: you need to replace Vue and @vitejs/plugin-vue by React and @vitejs/plugin-react.

**Comment by soulsam480** [comment]
Hii, is it possible for you to review after I complete the port ?

**Comment by cawa-93** [comment]
I didn't work with the React. So, my review will be useless.

**Comment by soulsam480** [comment]
The react port can be found here https://github.com/soulsam480/vite-electron-react-starter . The renderer process is working fine with react 17.x . Only known issue is it doesn't spawn the electron process inside WSL ubuntu environment. Works fine in windows 10. Also there may be some unused files and code as the port was done on the original repo. Thanks for your work @cawa-93 .

**Comment by cawa-93** [comment]
Find yet another useful forks:
https://useful-forks.github.io/?repo=cawa-93%2Fvite-electron-builder
